Entities/ThrownDevice.cpp

```diff
@@ -106,6 +113,21 @@ namespace RTE {
 		if (!m_Activated) {
 			m_ActivationTimer.Reset();
 			m_ActivationSound.Play(m_Pos);
+			// Launch striker lever, if there is one.
+			MovableObject *pStrikerLever = m_StrikerLever ? dynamic_cast<MovableObject *>(m_StrikerLever->Clone()) : 0;
```

Comment:
1. No need for a junk comment, we're trying to move away from those
2. The `p` thing in naming is from data's old code. Just name it `strikerLever`
3. This should be `nullptr` at the end. We don't use `0` for null pointers, that's old data code.